DFE-Digital/Dfe.Academies.External
# Moved trust into a subfolder and added trust consent page

Resolves [#104657](https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/104657)

## Type of change


## Expected behaviour


Trust consent page



## Steps to reproduce issue (if relevant)


1.
2.
3.
4.

## Steps to test this PR

1. Go to change a trust
2. Check you can upload a document from the trust consent page and it displays on the summary page
3.
4.

## Prerequisites
